RoosterPlanner.UI/src/app/pages/task/task.component.ts

```diff
@@ -53,9 +53,9 @@ export class TaskComponent implements OnInit {
     });
     dialogRef.disableClose = true;
     dialogRef.afterClosed().subscribe(result => {
-      if (result.status === 200) {
-        this.toastr.success(result.body.name + " is gewijzigd")
-        this.task = result.body;
+      if (result) {
```

Comment:
Waarom hier geen: if (result.ok)

Reply:
ipv de result.status ===200? 
Dat lijkt mij inderdaad een betere manier en zal ik meenemen in de globale refactorslag.

Overigens heb ik dat nu in task.service.ts zitten en wordt op regel 56 alleen gecontroleerd of het resultaat (type='task') niet null of undefined is

Reply:
Ja inderdaad ipv de result.status === 200
Prima om het mee te nemen in globale refactorslag